pandas/tests/frame/methods/test_sort_values.py

```diff
@@ -219,25 +219,30 @@ def test_sort_values_stable_descending_sort(self):
 
     def test_sort_values_stable_descending_multicolumn_sort(self):
```

Comment:
we should be able to parametrize this (I'd say on `expected, ascending, na_position`)

Reply:
Yes, definitely. For now, I've added all possible combinations (i.e. 8 parameter sets). If that's over the top, I can also remove some as some of the other tests also already test some of the combinations.